.github/workflows/end2end-test.yml

```diff
@@ -2,8 +2,18 @@ name: End-to-End Tests
 
 on:
   pull_request:
+    paths:
+    - 'lib/**'
+    - 'packages/**'
```

Comment:
I don't think these are enough, I'd rather just keep the exclusions. changes to package.json could be impactful, to gutenberg.php and probably other things I'm missing.

Reply:
ok, I didn't know if exclusion only would work, I like that better 👍 
Updated to reflect that.